tests/test_association.py

```diff
@@ -132,10 +131,10 @@ def createTestSources(nSources=5,
     for srcIdx in range(nSources):
         src = sources.addNew()
         src['id'] = srcIdx + startId
-        coord = Coord(sourceLocsDeg[srcIdx][0] * afwGeom.degrees,
-                      sourceLocsDeg[srcIdx][1] * afwGeom.degrees)
+        coord = afwGeom.SpherePoint(sourceLocsDeg[srcIdx][0],
+                                    sourceLocsDeg[srcIdx][1], afwGeom.degrees)
         if scatterArcsec > 0.0:
-            coord.offset(
+            coord = coord.offset(
```

Comment:
I suspect there need to be parallel changes in `ap_association`. See also [DM-11868](https://jira.lsstcorp.org/browse/DM-11868).

More generally, are we required to keep code up to date in `lsst-dm`? That seems like a difficult problem, especially when the packages in question are not part of CI.

Reply:
I already updated `ap_association` and `ap_pipe`. I was not required to do so, but I had them checked out and it was easy to do. If you like, I will leave merging of those three packages to you.

Reply:
No, it's fine. I must have missed the notification for `ap_association`.